runtime/gc_modron_startup/mminit.cpp

```diff
@@ -2717,7 +2717,12 @@ gcInitializeDefaults(J9JavaVM* vm)
 			/* If running jitted, it must be on supported h/w */
 			J9ProcessorDesc  processorDesc;
 			j9sysinfo_get_processor_description(&processorDesc);
-			if (j9sysinfo_processor_has_feature(&processorDesc, J9PORT_S390_FEATURE_GUARDED_STORAGE) && j9sysinfo_processor_has_feature(&processorDesc, J9PORT_S390_FEATURE_SIDE_EFFECT_ACCESS)) {
+			if ((j9sysinfo_processor_has_feature(&processorDesc, J9PORT_S390_FEATURE_GUARDED_STORAGE) &&
+					j9sysinfo_processor_has_feature(&processorDesc, J9PORT_S390_FEATURE_SIDE_EFFECT_ACCESS))
+#if defined(S390) || defined(J9ZOS390)
```

Comment:
Would it be better to have this check somewhere else and remove it here (have it false for other platforms)?

Reply:
moved the #ifdef check to mmparseXXgc.cpp